docs/phases/07-phase-backend-foundation.md

```
@@ -0,0 +1,411 @@
+---
+
+## API認証（シンプル版）
+
+MVP段階ではシンプルなAPIキー認証を使用:
+
+```typescript
+const API_KEY_HEADER = "X-API-Key";
+const VALID_API_KEY = "tempo-ai-mobile-app-key-v1";
+```
+
+**注意**: 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)